package.json

```diff
@@ -25,7 +25,7 @@
     "url": "https://github.com/OpenSTFoundation/mosaic.js/issues"
   },
   "scripts": {
-    "test": "mocha --recursive --timeout 5000",
+    "test": "MOSAIC_LOG_LEVEL=0 mocha --recursive --timeout 5000",
```

Comment:
What log messages does this suppress?

Reply:
Suppressing the facilitator logs for unit tests.